packages/cli/generators/relation/index.js

```diff
@@ -118,6 +119,11 @@ module.exports = class RelationGenerator extends ArtifactGenerator {
           utils.camelCase(this.artifactInfo.destinationModel),
         );
         break;
+      case relationUtils.relationType.hasOne:
+        defaultRelationName =
```

Comment:
`defaultRelation =` should be `utils.camelCase(this.artifactInfo.destinationModel)`